### PR TITLE
Fix broken url in docs

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary

Easy navigation and easy reading for others

### CODE REVIEW

**Comment by Conduitry** [comment]
This should instead be pointing to `app/App.html` at a suitably old commit on `master`. The only reason that this file still exists in the `page-transitions` branch is that that branch (and the other cookbook-y branches) is abandoned and out of date.

**Comment by saintmalik** [comment]
> This should instead be pointing to `app/App.html` at a suitably old commit on `master`. The only reason that this file still exists in the `page-transitions` branch is that that branch (and the other cookbook-y branches) is abandoned and out of date.

Okay, but how do I locate the old commit, or should I close the pull request.

File: site/content/migrating/01-migrating.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ The `4xx.html` and `5xx.html` error pages have been replaced with a single page,
 
 In earlier versions, each page was a completely standalone component. Upon navigation, the entire page would be torn down and a new one created. Typically, each page would import a shared `<Layout>` component to achieve visual consistency.
 
-As of 0.12, this changes: we have a single `<App>` component, defined in `app/App.html`, which controls the rendering of the rest of the app. See [sapper-template](https://github.com/sveltejs/sapper-template/blob/master/app/App.html) for an example.
+As of 0.12, this changes: we have a single `<App>` component, defined in `app/App.html`, which controls the rendering of the rest of the app. See [sapper-template](https://github.com/sveltejs/sapper-template/blob/page-transitions/app/App.html) for an example.
```
Comment (benmccann) [inline site/content/migrating/01-migrating.md:259]:
```suggestion
As of 0.12, this changes: we have a single `<App>` component, defined in `app/App.html`, which controls the rendering of the rest of the app. See [sapper-template](https://github.com/sveltejs/sapper-template/blob/365e8521ae6e1ffc5f1359342bb59c92a0014dca/app/App.html) for an example.
```

**Comment by saintmalik** [comment]
> > This should instead be pointing to `app/App.html` at a suitably old commit on `master`. The only reason that this file still exists in the `page-transitions` branch is that that branch (and the other cookbook-y branches) is abandoned and out of date.
> 
> Okay, but how do I locate the old commit, or should I close the pull request.

@benmccann thanks for the fix

**Pull request merged**